cogs/ca.py

```diff
@@ -523,8 +525,62 @@ async def sim(
                 rulename, rulefile, n_states, colors = await self.bot.pool.fetchrow('''
                   SELECT name, file, n_states, colors FROM rules WHERE name = $1::text
                 ''', rule)
-            except ValueError:  # not enough values to unpack
-                return await ctx.send('`Error: Rule not found`')
+            except TypeError:  # rule not found
+                # first, attempt to load rule form wiki
+                try:
+                    async with self.session.get(f"https://conwaylife.com/w/api.php?action=parse&format=json&page=RULE:{rule}") as resp:
+                        b = await resp.json()
+                        contents = b["parse"]["text"]["*"]
+                        d = contents.replace("<p>", "\n").replace("</p>", "\n")
+                        tmp = "@RULE" + "@RULE".join(d.split("@RULE")[1:])
```

Comment:
```suggestion
                    async with self.session.get(f"https://conwaylife.com/w/api.php?action=parse&format=json&prop=wikitext&page=RULE:{rule}") as resp:
                        b = await resp.json()
                        rulefile = b["parse"]["text"]["*"]
```
The query parameter `prop=wikitext` gets us the original, unparsed page text, which lets us avoid that string manipulation (phew!)

Reply:
Thanks! I didn't know about that!